### PR TITLE
Fix upgrade test from previous release

### PR DESCRIPTION
**Pull request by tiraboschi**
Perform upgrade test injecting the new CSV on
a new channel.
Correctly switch the upgrade channel during the upgrade test.

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

File: hack/upgrade-test.sh [hunk 1]
```diff
@@ -41,10 +41,19 @@
 # to verify that it is updated to the new operator image from 
 # the local registry.
 
-MAX_STEPS=8
+MAX_STEPS=9
 CUR_STEP=1
 RELEASE_DELTA="${RELEASE_DELTA:-1}"
 HCO_DEPLOYMENT_NAME=hco-operator
+HCO_NAMESPACE="kubevirt-hyperconverged"
```
Comment (nunnatsa) [inline hack/upgrade-test.sh:48]: missing 
`HCO_CATALOG_NAMESPACE=openshift-marketplace`

?

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

File: hack/upgrade-test.sh (hunk 1 above)
Comment (tiraboschi) [reply hack/upgrade-test.sh:48]: it's in hack/upgrade-openshiftci-config

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Pull request merged**